pom.xml

```diff
@@ -258,4 +151,122 @@
     </plugins>
   </build>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>26.0-jre</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+      <version>1.10</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-math3</artifactId>
+      <version>3.6.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.accumulo</groupId>
+      <artifactId>accumulo-client-mapreduce</artifactId>
+      <version>${accumulo.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.accumulo</groupId>
+      <artifactId>accumulo-core</artifactId>
+      <version>${accumulo.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
It would be really nice to set the scope of this to `test` because mini brings a lot of Accumulo and Hadoop server side cruft into the shaded jar.  I tried doing this and ran into a problem with Replication test using Accumulo server side code.  This mini dep brings in that server side code.  For local testing I deleted the Replication test code and was able to build a shaded jar that had much less warnings.